File: src/main/java/xyz/nietongxue/serviceql/BookRepository.java
package xyz.nietongxue.serviceql;

import org.springframework.stereotype.Component;

import java.util.ArrayList;
import java.util.List;



@Component
public class BookRepository {
    public List<Book> findAll() {
        List<Book> books = new ArrayList<Book>();
        books.add(new Book(1, "Jason's book", 1));
        return books;
    }
}

File: src/main/kotlin/xyz/nietongxue/serviceql/book/BookRepository.kt
package xyz.nietongxue.serviceql.book

import org.springframework.stereotype.Component

import java.util.ArrayList


@Component
class BookRepository {
    fun findAll(): List<Book> {
        val books = ArrayList<Book>()
        books.add(Book(1, "Jason's book", 1))
        return books
    }
}
